source/Company.py

```diff
@@ -1,53 +1,184 @@
+# Company Class exercise
+
+def contains_number(value):
+    '''
+    A function which checks if a value includes digits
+    '''
+    if True in [char.isdigit() for char in value]:
+        return True
+    return False
```

Comment:
This implementation works, but it is very clumsy. try using a simple for loop for this and return false once a char is found. It will be more readable and result in better preformance.

Reply:
To be honset, I have no idea what do those functions do.. it still looks a little clumsy to me, but ill let you decide if you know how it works.

Reply:
Ok. If you have a better idea implementation, we can talk about it.